---
layout: post
title: SIRT7 links H3K18 deacetylation to maintenance of oncogenic transformation
date: 2019-08-07
tag: journal article
---

### title :
**SIRT7 links H3K18 deacetylation to maintenance of oncogenic transformation**

  Wei Li and Katrin F. Chua
  Nature. 2012 July 5; 487(7405): 114–118. doi:10.1038/nature11043.

### key words:
- SIRT7
- H3K18Ac
- transcription factor ELK4
- viral oncoprotein E1A

### summary :
1. SIRT7 is a chromatin-associated H3K18Ac-specific deacetylase
2. SIRT7 binds to gene promoters and couples H3K18 deacetylation to transcriptional repression
3. SIRT7 is stabilized at target promoters by interaction with the ETS family transcription factor ELK4
4. SIRT7 depletion reverses cancer cell phenotypes and inhibits tumour growth in vivo

### notes :
**the first step is to find SIRT7 deacetylation sites**
>* SIRT7 might function at chromatin→mass spectrometry screening
>* SIRT7 exhibited highly specific deacetylase activity on peptides containing H3K18Ac.  SIRT1 has a broader substrate spectrum .
>* SIRT7 also exhibited robust and specific NAD+ -dependent H3K18Ac-deacetylase activity on full-length histone H3 in purified poly-nucleosomes .
>* This activity was abolished by substitution of a conserved histidine residue (H187→Y) in the predicted catalytic domain of SIRT7 and by the general sirtuin inhibitor nicotinamide.
>* Selective H3K18Ac deacetylation was also observed in cells following overexpression of the wild-type, but not mutant, SIRT7 protein.
>* An unbiased proteomic approach using quantitative mass spectrometry independently demonstrated that SIRT7 overexpression induces a dramatic depletion of H3K18Ac in cells.

**investigate how SIRT7 contacts with H3K18Ac**
>* The genome-wide occupancy of SIRT7 by ChIP-sequencing. Promoter enrichment and target gene analysis of SIRT7 ChIP-sequencing.
>* SIRT7 ChIP-sequencing peaks correspond to 241 protein-coding genes.
>* Using ChIP-qPCR, we confirmed the binding of SIRT7 at several of the identified promoters and alidated the specificity of the ChIP signals by siRNA-mediated depletion of SIRT7.
>* Functional categorization of the SIRT7-bound genes revealed strong enrichment for factors involved in RNA processing, protein translation and cellular macromolecule metabolism, with diverse links to tumour suppressive activities.
>* SIRT7 deacetylates H3K18Ac at target promoters.SIRT7-depletion led to hyperacetylation of H3K18 at the promoters of the RPS20, RPS7, RPS14, NME1, and COPS2 genes, but not multiple negative control promoters.
>* Consistent with this locus specificity, global H3K18Ac levels were not affected by SIRT7 depletion .SIRT7 depletion does not increase global levels of H3K18Ac.
>* SIRT7 depletion induces transcriptional activation of target genes.SIRT7-occupied genes are up-regulated upon SIRT7 depletion.ChIP-seq profiles.HDAC1 does not regulate several SIRT7 target genes.HDAC1 can also deacetylate H3K18Ac.

**detect possible transcription factor by using bioinformatic strategy**
>* The SIRT7 consensus motif was most similar to the DNA sequence recognized by the ETS protein ELK4 .In the JASPAR CORE database.
>* ELK4 binds several of the SIRT7 target promoters that contain the ETS consensus motif (NME1 and COPS2), but not promoters lacking the motif (RPS20 or GAPDH)
>* in co-immunoprecipitation experiments, we found that SIRT7 interacts physically with ELK4, but not with two other ETS proteins, ELK1 and GABPα .
>* Depletion of ELK4 led to a partial but significant decrease in SIRT7 occupancy at the NME1 and COPS2 promoters but not the RPS20 promoter and did not alter global levels of SIRT7 at chromatin.
>* ELK4 knockdown led to elevated H3K18Ac levels at the NME1 and COPS2 promoters, but not at promoters lacking the ETS motif .
>* knockdown of ELK4, but not ELK1 or GABPα, was sufficient to impair gene repression induced by overexpression of SIRT7 .
>* SIRT7 is over-expressed in human tumour samples.

**connect to SIRT7 depletion phenotype**
>* SIRT7 depletion in HT1080 and U2OS cells severely impaired both anchorage-independent cellular growth in soft agar and proliferation in low serum, two important hallmarks of transformed cells
>* SIRT7-depletion were associated with both increased cell death and altered cell-cycle progression
>* Importantly, functional reconstitution assays revealed that the catalytic activity of SIRT7 is necessary for maintenance of the cancer cell-specific growth properties
>* simultaneous expression of SIRT7 and ELK4 had a synergistic effect on maintenance of the transformed phenotype of these cells
>* SIRT7 depletion in HT1080 cells severely inhibited this E1A-dependent reduction of H3K18Ac
>* SIRT7 depletion impairs E1A-mediated loss of contact inhibition in primary IMR90 fibroblasts
>* tumour formation was severely impaired by SIRT7 depletion

### figures&tables :
<img src='https://i.loli.net/2019/08/07/tPSlCkfy4NE5dVx.png'>
Changes in global histone acetylation levels in SIRT7 overexpressing versus control 293T cells, determined by quantitative mass spectrometry.

<img src='https://i.loli.net/2019/08/07/zDlmteRfCbrKsT9.png'>
SIRT7 is stabilized at target promoters by interaction with the ETS family transcription factor ELK4
